pkg/types/rpm/v0.0.1/entry.go

```diff
@@ -104,6 +106,19 @@ func (v V001Entry) IndexKeys() []string {
 		}
 	}
 
+	// if this is a certificate, add email addresses to result
```

Comment:
RPMs are only signed with PGP so I think we would need to consider how to extract it from that key. Also see typecast from above. 

Should we consider extending the base class for public key to have a method for this and then simply call it here?

Reply:
I like extending the base class! I'll do that and try to support pgp -- will try to get the email out of the User ID (and hope it has an angle bracket enclosed email)

Reply:
Done, I got the PGP User ID Email (https://github.com/golang/crypto/blob/eec23a3978ad/openpgp/packet/userid.go#L16), but in case it's in Name, I don't parse it.